### DISCUSSION TITLE
Delivery pincode checker , Ajax product search , order tracking details in timeline below order order details

### DISCUSSION THREAD

**Discussion by anikesh002** [inspireui/support#5493] [Working on Features]
This is the most important features that all e-commerce must have 
1. Pincode Delivery checker & estimate delivery date .
2. Ajax Search for instant product search result 
3. Order tracking details in  timeline below order details like other e-commerce website .

**Comment by tryabp** [comment]
I think famous codecanyon plugin : AJAX SEARCH PRO integration will do best job for search module. (https://codecanyon.net/item/ajax-search-pro-for-wordpress-live-search-plugin/3357410) 
I hope fluxstore team consider this.

**Reply by osman-kocak** [reply]
we really need a good search
